### PR TITLE
Fix Java version for Ubuntu 20.04 setup script

### PR DESCRIPTION
**Pull request by julianoes**
Check commits to review.

### CODE REVIEW

**Review by dagar — APPROVED** [review]

**Comment by julianoes** [comment]
What's up with CI?

**Review by MaEtUgR — APPROVED** [review]
Nice, just ran into this yesterday and wanted to create a pr now.
Luckily I first found yours and you even fix the silent failure issue 👍 

I tested this on my Ubuntu 20.04 install from yesterday removing the openjdk I installed, having jMAVsim fail with the presumably preinstalled version, running the script with your changes, now works as expected.

File: Tools/setup/ubuntu.sh [hunk 1]
```diff
@@ -1,5 +1,7 @@
 #! /usr/bin/env bash
 
+set -e
```
Comment (MaEtUgR) [inline Tools/setup/ubuntu.sh:3]: Maybe a comment why this is useful might save some people some time...

**Comment by dagar** [comment]
> What's up with CI?

Multiple PRs went in and the combination was over flash only once merged together. I'll have it fixed soon.

File: Tools/setup/ubuntu.sh (hunk 1 above)
Comment (julianoes) [reply Tools/setup/ubuntu.sh:3]: https://www.google.com/search?&q=%22set+-e%22 :trollface:

**Pull request merged**